schnorr_fun/src/lib.rs

```diff
@@ -22,6 +22,8 @@ extern crate serde_crate as serde;
 pub use secp256kfun as fun;
 pub use secp256kfun::nonce;
 
+/// Binonce for Musig and FROST
+pub mod binonce;
```

Comment:
I think make this private and re-export stuff from it in both `musig` and `frost`. i.e. internally we are re-using it but people using the API can't tell that and always import the one from the API they are using.